### ISSUE TITLE
Spectrogram fine grained-control options

### ISSUE THREAD

**Issue by atruskie** [QutBioacoustics/baw-client#300]
We've has multiple requests to allow more options for dealing with spectrograms.

On the _listen_ page, add options to:

- [ ] Change the sample rate of the current audio and spectrogram
- [ ] Change the window size of the spectrogram FFT
- [ ] Change the selected channel for audio and spectrogram
- [ ] Change the size of the spectrogram (just image stretching, and I think, for now, just a 2x Zoom)
- [ ] Change the colour of the spectrogram

@JessCappadonna, @KarlinaInd, @tsheringde: does this represent the breadth of enhancements asked for?

**Comment by JessCappadonna** [comment]
Yes, @atruskie I think those capture all of the specific requests from my interviews with people having past experiences with other acoustic software. I have other citizen science engagement ideas, which need further consideration before adding to the list or creating a knew thread at present.

**Comment by KarlinaInd** [comment]
To date I don't really have specific needs, although might have them in the future. Regardless, I think having those configurations will be great, for future analysis. Thanks

**Comment by atruskie** [comment]
This was requested by another user.

This feature is in our long-term plans but we haven't got it prioritised at the moment.

**Comment by debrastark** [comment]
Yes @atruskie all of those features would be great! Specifically, I would like to be able to change the colour, window size and timescale of the spectrogram. My frogs tend to have really short calls (0.3-0.5 seconds) which can be hard to see at the current 30 second scale. Viewing the spectrogram in colour is also a lot more useful for me as L. fallax tends to call at louder amplitudes and so the colour spectrograms makes the calls more visible when it overlaps with L. olongburensis. Window size is also an important feature for me as it increases the clarity of the calls. Thanks!
